.github/workflows/test.yml

```diff
@@ -62,6 +62,30 @@ jobs:
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
 
+  chromatic-deployment:
```

Comment:
Made chromatic it's own job. Should I make it a a step instead?

Reply:
This is right call. I think Percy was approved at tier 3, where as chromatic was approved at tier 2 since it has access to git history.